test/e2e/upload.test.js

```diff
@@ -1,15 +1,235 @@
-
 const { assert, driver } = require('vl-ui-core').Test.Setup;
 const VlUploadPage = require('./pages/vl-upload.page');
+const path = require('path');
+const Express = require("express");
+const Multer = require("multer");
```

Comment:
Eventueel al de bestanden globaal één keer ophalen zodat ze in al de testen gebruikt kunnen worden.

Reply:
hulp methode gemaakt om de een bestand te resolven